src/app/common/UpgradeCheck.spec.ts

```diff
@@ -1,29 +1,36 @@
 import { describe, expect, test } from '@jest/globals'
 import { flushPromises, mount } from '@vue/test-utils'
+import { container, TOKENS, service, injected } from '@/services'
+import Env from '@/services/env/Env'
 
 import UpgradeCheck from './UpgradeCheck.vue'
-import container from '@/services/container'
 
 function renderComponent() {
   return mount(UpgradeCheck)
 }
 
 describe('UpgradeCheck.vue', () => {
   test('renders snapshot', async () => {
-    const env = container.get('env')
-    container.set('env', {
-      var: (key: string) => {
+    container.capture?.()
```

Comment:
**Question**: What is this doing (same for the restore call)?

Reply:
`capture` saves the state of the container so you can change things so you can then call `restore` to put it back to how it was, perfect for before/after hooks.

https://brandi.js.org/reference/container#capture
https://brandi.js.org/reference/container#restore